src/devices/openxrheadset/OpenXrInterface.cpp

```diff
@@ -735,6 +762,28 @@ bool OpenXrInterface::prepareXrActions()
     return true;
 }
 
+bool OpenXrInterface::prepareHandTracking() {
+    // hand tracking
+    if (m_pimpl->hand_tracking_supported) {
```

Comment:
Nitpick. You can avoid an indentation level and return early if ``hand_tracking_supported`` is false, i.e.
```c++
if (!m_pimpl->hand_tracking_supported) {
    return true;
}
... rest of the code
```